### DISCUSSION TITLE
Typst: Show File Names of Code Blocks

### DISCUSSION THREAD

**Discussion by MathisC22** [quarto-dev/quarto-cli#9627] [Feature Requests]
### Description

File names of code cell blocks are not shown when using Typst as backend, whereas they do show when you use  LaTeX or HTML to render the file. That is, the following cell only shows the contents:
```` 
```{.python filename="matplotlib.py"}
import matplotlib.pyplot as plt
plt.plot([1,23,2,4])
plt.show()
```
```` 
Would be really neat if it would also work with Typst!

**Comment by cderv** [comment]
This is indeed not supported. We need to add support for it. Thanks !

* https://github.com/quarto-dev/quarto-cli/issues/9768
